From 71301ccf8aa54f7272e7ef8009402db622fe8cd9 Mon Sep 17 00:00:00 2001
From: Alexey Tikhonov <atikhono@redhat.com>
Date: Tue, 22 Jun 2021 10:29:44 +0200
Subject: [PATCH 16/16] KCM: removed unneeded assignment
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes following warning:
```
Error: CLANG_WARNING:
sssd-2.5.1/src/responder/kcm/kcm_renew.c:481:9: warning[deadcode.DeadStores]: Value stored to 'ret' is never read
 #  479|       ctx = talloc_zero(auth_data, struct kcm_renew_auth_ctx);
 #  480|       if (ctx == NULL) {
 #  481|->         ret = ENOMEM;
 #  482|           DEBUG(SSSDBG_FATAL_FAILURE, "Failed to allocate renew auth ctx\n");
 #  483|           return;
```

Reviewed-by: Pavel Březina <pbrezina@redhat.com>
---
 src/responder/kcm/kcm_renew.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/responder/kcm/kcm_renew.c b/src/responder/kcm/kcm_renew.c
index c619ed0a8de3b000db61866d21cb8b71764c0ac0..684d08be6affdf5e4acbd4b04b5a61f154c152cc 100644
--- a/src/responder/kcm/kcm_renew.c
+++ b/src/responder/kcm/kcm_renew.c
@@ -478,7 +478,6 @@ static void kcm_renew_tgt(struct tevent_context *ev,
 
     ctx = talloc_zero(auth_data, struct kcm_renew_auth_ctx);
     if (ctx == NULL) {
-        ret = ENOMEM;
         DEBUG(SSSDBG_FATAL_FAILURE, "Failed to allocate renew auth ctx\n");
         return;
     }
-- 
2.20.1

